justfile

```diff
@@ -99,7 +100,33 @@ rust-lld := if target-os != "windows" { " -Z gcc-ld=lld" } else { "" }
 # rust-lld.
 rustc-icf := if for-release != "" { " -C link-arg=-Wl,--icf=safe" } else { "" }
 
-cargo-build-args := (if for-release != "" { " --release" } else { "" }) + (if target != target-host { " --target " + target } else if cargo-buildstd != "" { " --target " + target } else { "" }) + (cargo-buildstd) + (if extra-build-args != "" { " " + extra-build-args } else { "" }) + (cargo-no-default-features) + (cargo-split-debuginfo) + (if cargo-features != "" { " --features " + cargo-features } else { "" }) + (win-arm64-ring16)
+target-glibc-ver-postfix := if glibc-version != "" {
+    if use-cargo-zigbuild != "" {
+        "." + glibc-version
+    } else {
+        ""
+    }
+} else {
+    ""
+}
+
+need-target := if target != target-host {
+    "need"
+} else if cargo-buildstd != "" {
+    "need"
+} else if target-glibc-ver-postfix != "" {
+    "need"
+} else {
+    ""
+}
+
+target-args := if need-target != "" {
+    "--target " + target + target-glibc-ver-postfix
+} else {
+    ""
+}
```

Comment:
If there's no disadvantage to using target when it's not needed, I prefer to just always specify it, as that makes the paths more predictable.

Reply:
That's a good idea!
I've adopted it.